README.md

```
@@ -41,17 +41,27 @@ bunx @vera-ai/cli install   # or: npx -y @vera-ai/cli install / uvx vera-ai inst
 ```
 
 **2. Set up and index** (pick one)
+
+Recommended: the Qwen preset via OpenRouter (best measured search quality, single API key):
 ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)